src/getting_started/README.md

```diff
@@ -1,13 +1,70 @@
-# Getting Started
+# Getting Started With Parsec
 
-Quickly get the Parsec service up and running to try it out! Choose the flavour that you would
-prefer:
+## Welcome!
 
-- [Quickstart for Linux on x86](linux_x86.md): running Parsec with Mbed Crypto natively on your
-   machine, the fastest
-- [Quickstart for openSUSE and SUSE](opensuse.md): running Parsec with the package from openSUSE and
-   SUSE distributions
-- [Quickstart using a Docker container](docker.md): running a Docker container with all providers
-   included, the most complete
+Welcome to the Parsec Getting Started guide. Take a look through the headings below and follow the
+links that best describe what you would like to do.
+
+## I Want to Check if Parsec is Running
```

Comment:
Minor: It's more of a personal preference (which you can ignore)
Would it be okay if `I Want to` is removed

Reply:
I worded it this way so that people could identify with the section(s) that are relevant to them.